wb-mqtt-serial-templates/config-wb-mir.json

```diff
@@ -118,6 +294,97 @@
                 "poll_interval": 0
             },
             {
+                "name": "Play from ROM8",
+                "reg_type": "coil",
+                "address": 5107,
+                "type": "pushbutton",
+                "poll_interval": 0
+            },
+            {
+                "name": "Play from ROM9",
+                "reg_type": "coil",
+                "address": 5108,
+                "type": "pushbutton",
+                "poll_interval": 0
+            },
+            {
+                "name": "Play from ROM10",
+                "reg_type": "coil",
+                "address": 5109,
+                "type": "pushbutton",
+                "poll_interval": 0
+            },
+            {
+                "name": "Play from ROM11",
+                "reg_type": "coil",
+                "address": 5110,
+                "type": "pushbutton",
+                "poll_interval": 0
+            },
+            {
+                "name": "Play from ROM12",
+                "reg_type": "coil",
+                "address": 5111,
+                "type": "pushbutton",
+                "poll_interval": 0
+            },
+            {
+                "name": "Play from ROM13",
+                "reg_type": "coil",
+                "address": 5112,
+                "type": "pushbutton",
+                "poll_interval": 0
+            },
+            {
+                "name": "Play from ROM14",
+                "reg_type": "coil",
+                "address": 5113,
+                "type": "pushbutton",
+                "poll_interval": 0
+            },
+            {
+                "name": "Play from ROM15",
+                "reg_type": "coil",
+                "address": 5114,
+                "type": "pushbutton",
+                "poll_interval": 0
+            },
+            {
+                "name": "Play from ROM16",
+                "reg_type": "coil",
+                "address": 5115,
+                "type": "pushbutton",
+                "poll_interval": 0
+            },
+            {
+                "name": "Play from ROM17",
+                "reg_type": "coil",
+                "address": 5116,
+                "type": "pushbutton",
+                "poll_interval": 0
+            },
+            {
+                "name": "Play from ROM18",
+                "reg_type": "coil",
+                "address": 5117,
+                "type": "pushbutton",
+                "poll_interval": 0
+            },
+            {
+                "name": "Play from ROM19",
+                "reg_type": "coil",
+                "address": 5118,
+                "type": "pushbutton",
+                "poll_interval": 0
+            },
+            {
+                "name": "Play from ROM20",
+                "reg_type": "coil",
+                "address": 5119,
+                "type": "pushbutton",
+                "poll_interval": 0

```

Comment:
Could you please remove "poll_interval": 0 here and everywhere else in the template?
It was added by mistake